scripts/jest/test-framework-setup.js

```diff
@@ -58,3 +58,26 @@ env.afterEach(() => {
   expect(console.error).toBeReset();
   expect(console.error).toNotHaveBeenCalled();
 });
+
+function wrapDevMatcher(obj, name) {
+  const original = obj[name];
+  obj[name] = function devMatcher() {
+    try {
+      original.apply(this, arguments);
+    } catch (e) {
```

Comment:
They all throw now except toMatchSnapshot. Also the matchers don't even come from jasmine any more – they're from "jest-matchers".